aws/request_1_8_test.go

```diff
@@ -1,84 +0,0 @@
-// +build go1.8
-
-package aws_test
-
-import (
-	"bytes"
-	"io"
-	"net/http"
-	"net/http/httptest"
-	"strings"
-	"testing"
-
-	"github.com/aws/aws-sdk-go-v2/aws"
-	"github.com/aws/aws-sdk-go-v2/internal/awstesting"
-	"github.com/aws/aws-sdk-go-v2/internal/awstesting/unit"
-)
-
-func TestResetBody_WithEmptyBody(t *testing.T) {
-	r := aws.Request{
-		HTTPRequest: &http.Request{},
-	}
-
-	reader := strings.NewReader("")
-	r.Body = reader
-
-	r.ResetBody()
-
-	if a, e := r.HTTPRequest.Body, http.NoBody; a != e {
-		t.Errorf("expected request body to be set to reader, got %#v",
-			r.HTTPRequest.Body)
-	}
-}
-
-func TestRequest_FollowPUTRedirects(t *testing.T) {
```

Comment:
Want to make sure these tests are not deleted. I think they are still valid for the v2 SDK.